deploy-manage/manage-connectors.md

```diff
@@ -2,25 +2,98 @@
+::::{note}
+This page is about Kibana connectors that integrate with services like generative AI model providers. If you’re looking for Search connectors that synchronize third-party data into {{es}}, refer to [Connector clients](https://www.elastic.co/guide/en/serverless/current/elasticsearch-ingest-data-through-integrations-connector-client.html).
 
-% Use migrated content from existing pages that map to this page:
+::::
 
-% - [ ] ./raw-migrated-files/kibana/kibana/action-types.md
-%      Notes: 28 children
-% - [ ] ./raw-migrated-files/docs-content/serverless/action-connectors.md
+## Required permissions [_required_permissions_2]
 
-$$$connector-management$$$
+Access to connectors is granted based on your privileges to alerting-enabled features. For more information, go to [Security](../explore-analyze/alerts-cases/alerts/alerting-setup.md#alerting-security).
+
+## Connector networking configuration [_connector_networking_configuration]
+
+Use the [action configuration settings](https://www.elastic.co/guide/en/kibana/current/alert-action-settings-kb.html#action-settings) to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+## Connector list [connectors-list]
+
+In **{{stack-manage-app}} > {{connectors-ui}}**, you can find a list of the connectors in the current space. You can use the search bar to find specific connectors by name and type. The **Type** dropdown also enables you to filter to a subset of connector types.
+
+:::{image} ../images/kibana-connector-filter-by-type.png
+:alt: Filtering the connector list by types of connectors
+:class: screenshot
+:::
+
+You can delete individual connectors using the trash icon. Alternatively, select multiple connectors and delete them in bulk using the **Delete** button.
+
+:::{image} ../images/kibana-connector-delete.png
+:alt: Deleting connectors individually or in bulk
+:class: screenshot
+:::
+
+::::{note}
+You can delete a connector even if there are still actions referencing it. When this happens the action will fail to run and errors appear in the {{kib}} logs.
+
+::::
+
+## Creating a new connector [creating-new-connector]
+
+New connectors can be created with the **Create connector** button, which guides you to select the type of connector and configure its properties. For a full list of available connectors, see [Available connectors](asciidocalypse://docs/kibana/docs/reference/connectors-kibana/connectors-kibana.md)
+
+::::{note}
+Some connector types are paid commercial features, while others are free. For a comparison of the Elastic subscription levels, go to [the subscription page](https://www.elastic.co/subscriptions).
+::::
+
+:::{image} ../images/kibana-connector-select-type.png
+:alt: Connector select type
+:class: screenshot
+:::
```

Comment:
this image is vv large - can we use the `width` attribute to resize? 

<img width="826" alt="image" src="https://github.com/user-attachments/assets/054fc3d3-1654-49d8-bbd1-fdbdf11173b3" />